server/server.js

```diff
@@ -1,5 +1,6 @@
 /* eslint-disable no-console */
 /* eslint-disable camelcase */
+/* istanbul ignore file */
```

Comment:
??? what is this boi

Reply:
it's been declared on the commit message. Ignore this file for test coverage